modules/machine-user-infra-machines-pxe.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
+
+[id="machine-user-infra-machines-pxe_{context}"]
+= Creating more {op-system-first} machines by PXE or iPXE booting
+
+You can create more compute machines for your bare metal cluster by using
+PXE or iPXE booting.
+
+.Prerequisites
+
+* Obtain the URL of the Ignition config file for the compute machines for your
+cluster. You uploaded this file to your HTTP server during installation.
+* Obtain the URLs of the {op-system} ISO image, compressed metal BIOS, `kernel`
+and `initramfs` files that you uploaded to your HTTP server during cluster
+installation.
+* Access to the PXE booting infrastructure that you used to create the machines
+for your {product-title} cluster during installation. The machines must boot
+from their local disks after {op-system} is installed on them.
+* If you use UEFI, you must have access to the `grub.conf` file that you modified
+during {product-title} installation.
+
+.Procedure
+
+. Confirm that your PXE or iPXE installation for the {op-system} images is
+correct.
+
+** For PXE:
++
+----
+DEFAULT pxeboot
+TIMEOUT 20
+PROMPT 0
+LABEL pxeboot
+    KERNEL http://<HTTP_server>/rhcos-<version>-<architecture>-installer-kernel <1>
+    APPEND ip=dhcp rd.neednet=1 initrd=http://<HTTP_server>/rhcos-<version>-<architecture>-installer-initramfs.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-<architecture>-metal-bios.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <2> <3>
+----
+<1> Specify the location of the `kernel` file that you uploaded to your HTTP
+server.
+<2> If you use multiple NICs, specify a single interface in the `ip` option.
+For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
+<3> Specify locations of the {op-system} files that you uploaded to your
+HTTP server. The `initrd` parameter value is the location of the `initramfs` file,
+the `coreos.inst.image_url` parameter value is the location of the compressed
+metal BIOS file, and the `coreos.inst.ignition_url` parameter value is the
+location of the bootstrap Ignition config file.
```

Comment:
We need the worker.ign file, not the bootstrap one. change "location of the bootstrap Ignition config file" to "location of the worker Ignition config file"